samples/cntd/file_policy_amp_threatgrid.yaml

```diff
@@ -0,0 +1,269 @@
+######
+# IMP: write in reverse order of dependencies to see expected behaviour(to avoid missing CRD errors) i.e. independent CRDs should come first
+# example: ThreatConfig CRD should come after ThreatPolicy and EventConfiguration CRDs as it is dependent on them
+######
+
+# CRDs to deploy File Policy, AMP and ThreatGrid
+
+# A File Policy is a set of configurations that the system uses to perform 
+# Malware Protection and File Control. This ensures that before the system 
+# passes a file in traffic that matches a Threat Policy rule’s conditions, 
+# it first inspects the file. 
+
+# The Advanced Malware Protection cloud or AMP is a Cisco-hosted server 
+# that uses big data analytics and continuous analysis to provide 
+# intelligence that the system uses to detect and block malware on 
+# your network. The AMP cloud provides dispositions for possible malware 
+# detected in network traffic by managed devices, as well as data updates 
+# for local malware analysis and file pre-classification.
+
+apiVersion: cntd.cisco.com/v1
+kind: IntrusionPolicy
+metadata:
+  name: my-ips-policy
+  namespace: sfcn-system
+spec:
+  type: balanced-ips
+  mode: Inline
+  addedRules:
+    - gid: 1
+      sid: 1234
+      action: "Block"
+  disabledRules:
+    - gid: 1
+      sid: 1235
+  addedRuleGroups:
+    f58ecfed-c8ec-5ffe-bf68-d33324e22289:
+      securityLevel: 2
+  excludedRuleGroups:
+    - d7011cb1-b435-593d-86b1-1b9ac2481fb1
+  treatBlockAsReject: false
+---
+apiVersion: cntd.cisco.com/v1
+kind: FilePolicy
+metadata:
+  name: my-file-policy
+  namespace: sfcn-system
+spec:
+  firstTimeAnalysis: true
+  threatScore: 60
+  maxArchiveDepth: 2
+  fileRules:
+    - id:
+        id: "test-sandbox-rule"
+      fileTypes:
```

Comment:
Could we provide info on what these filetypes refer to?

Reply:
Added description in Appendix.md